evm/utils/headers.py

```diff
@@ -74,11 +76,11 @@ def compute_gas_limit(parent_header, gas_limit_floor):
 
 
 def generate_header_from_parent_header(
-        compute_difficulty_fn,
-        parent_header,
-        coinbase,
-        timestamp=None,
-        extra_data=b''):
+        compute_difficulty_fn: Callable,
```

Comment:
Care to expand this to include the arguments it takes?

Reply:
yep, let me try. I think this should be `Callable[[BlockHeader, int], int]`, right?